account_cutoff_picking/migrations/16.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if not openupgrade.column_exists(
+        env.cr, "res_company", "default_cutoff_picking_interval_days"
+    ):
+        openupgrade.rename_fields(
```

Comment:
wondering: shouldn't we move fields to the new module name?

Reply:
what do you mean?
this what is done in this migration script for `res.company`
I suppose it's needed only for this field, because fresh installation won't cause any problems.
And this is proposed to migrate from v15 to v16 on existing projects.

Reply:
there are only two fields introduced in this module:
`default_cutoff_accrual_picking_interval_days` -> `default_cutoff_picking_interval_days`:needs to be renamed
`picking_interval_days` name stays the same
https://github.com/OCA/account-closing/blob/f0fde8029c384c72f50f002ecde43de090dce697/account_cutoff_accrual_picking/models/account_cutoff.py#L15